lib/ClangImporter/ClangImporter.cpp

```diff
@@ -574,6 +574,10 @@ void importer::getNormalInvocationArguments(
     }
   }
 
+#ifdef SWIFT_ENABLE_EXPERIMENTAL_POINTER_BOUNDS
+  invokationsArgStrs.push_back("-fexperimental-bounds-safety-attributes");
```

Comment:
This is going to be part of the bounds safety upstreaming effort

Reply:
Ah, okay. So that's an llvm-project change that allows the use of the `counted_by`, `sized_by`, etc. Clang attributes in headers outside of `-fbounds-safety`, right?

Reply:
Indeed!